public/js/view.js

```diff
@@ -0,0 +1,100 @@
+$(document).ready(() => {
```

Comment:
`$(document).ready` should be as small as possible and all of the other functions should be separated out.  `$(document).ready` should just contain the code required to launch your application (load all relevant data and set up anything on the page) when the page loads